orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/SomaticFindingsChapter.java

```diff
@@ -223,12 +226,14 @@ private void addFusions(@NotNull Document document)
     {
         String driverFusionsTitle = "Driver fusions";
         String nonDriverFusionsTitle = "Other potentially interesting fusions";
+        String inFrameFusionsTitle = "Potentially interesting in-frame fusions in case no high drivers detected";
```

Comment:
I would make the naming more clear. Previously we had "drivers" and "non-drivers" but now there is a 3rd category (which are also non-driver!)